src/OpenTelemetry/BaseExporter.cs

```diff
@@ -74,7 +74,7 @@ public bool Shutdown(int timeoutMilliseconds = Timeout.Infinite)
         {
             if (timeoutMilliseconds < 0 && timeoutMilliseconds != Timeout.Infinite)
             {
-                throw new ArgumentOutOfRangeException(nameof(timeoutMilliseconds));
+                throw new ArgumentOutOfRangeException(nameof(timeoutMilliseconds), timeoutMilliseconds, $"timeout milliseconds should not be below 0 or infinite, but was: {timeoutMilliseconds}");
```

Comment:
`Timeout.Infinite` is actually `-1`. Here the correct message can be taken from Line:67.

Reply:
Fixed this in latests commit.